app/templates/components/forms/wizard/sessions-speakers-step.hbs

```diff
@@ -252,24 +264,36 @@
               </th>
             </tr>
             <tr>
-              <th class="right aligned">{{t 'Option'}}</th>
-              <th class="center aligned">{{t 'Include'}}</th>
-              <th class="center aligned">{{t 'Require'}}</th>
+              {{#if device.isMobile}}
+                <th class="center aligned">
+                  {{t 'Options'}}
+                </th>
+              {{else}}
+                <th class="right aligned">
+                  {{t 'Option'}}
+                </th>
+                <th class="center aligned">
+                  {{t 'Include'}}
+                </th>
+                <th class="center aligned">
+                  {{t 'Require'}}
+                </th>
+              {{/if}}
             </tr>
           </thead>
           <tbody>
             {{#each sessionsSpeakers.customForm.session as |field|}}
               <tr class="{{if field.isIncluded 'positive'}}">
-                <td class="right aligned"><label class="{{if field.isFixed 'required'}}">{{field.name}}</label></td>
+                <td class="{{if device.isMobile 'center' 'right'}} aligned">
+                  <label class="{{if field.isFixed 'required'}}">
+                    {{field.name}}
+                  </label>
+                </td>
                 <td class="center aligned">
-                  <div class="ui slider checkbox">
-                    {{input type='checkbox' checked=field.isIncluded disabled=field.isFixed}}
-                  </div>
+                  {{ui-checkbox class='slider' checked=field.isIncluded disabled=field.isFixed label=(if device.isMobile (t 'Include'))}}
                 </td>
                 <td class="center aligned">
-                  <div class="ui slider checkbox">
-                    {{input type='checkbox' checked=field.isRequired disabled=field.isFixed}}
-                  </div>
+                  {{ui-checkbox class='slider' checked=field.isRequired disabled=field.isFixed label=(if device.isMobile (t 'Require'))}}
```

Comment:
mutate `field. isRequired` on change. This another thing that is required when `ui-checkbox` is used.